src/components/menu/menu.css

```diff
@@ -44,3 +46,54 @@
 .menu-section {
     border-top: 1px solid $ui-black-transparent;
 }
+
+.submenu {
+    display: none;
+    position: absolute;
+    top: -1px;
+    min-width: 186px;
+    max-width: 260px;
+    height: max-content;
+    overflow: hidden;
+    border-radius: 8px;
+}
+
+.submenu.right {
+    left: 100%;
+}
+
+.submenu.left {
+    right: 100%;
+}
+
+.submenu > .menu {
+    position: relative;
+    overflow-y: auto;
+    overflow-x: hidden;
+    scrollbar-color: $ui-black-transparent $looks-secondary;
+    border-radius: 8px;
+}
+
+.menu-item.expanded > .submenu {
+    display: block;
+}
+
+:not(:has(> .menu-item.expanded)) > .menu-item:hover > .submenu {
+    display: block;
+}
```

Comment:
This styling is meant to expand the submenus when clicked or hovered. [:has() support](https://caniuse.com/css-has) does not cover all of our supported browsers. (Firefox is the obvious omission.) It wouldn't be impossible to circumvent this, perhaps adding a class to the parent of `.menu-item` based on whether any children are expanded. However, the hovering behavior is not critical. Users can still navigate the submenus by clicking. I could be convinced that identical behavior across browsers is more important than code simplicity...